prow/cluster/velodrome/velodrome_service.yaml

```diff
@@ -13,10 +27,12 @@ spec:
 kind: Endpoints
 apiVersion: v1
 metadata:
-  name: velodrome
+  labels:
+    app.kubernetes.io/part-of: velodrome
+  name: velodrome-external
```

Comment:
Wouldn't velodrome-internal make more sense here?

Reply:
No it is an external IP.

Reply:
Then what is the point of this PR?

Reply:
I think that is clear from the title?

Reply:
What is the difference between velodrome and velodrome-external then? Why do both of these exist?

Reply:
The internal service (i.e. `veldorome`) is an intermediate hop for control over load balancing, cert provisioning, and redirecting before proxying to external service (i.e. `velodrome-external`).

User -> Internet -> Velodrome(internal service) -> Nginx proxy -> Velodrome(external service).